tests/PayloadTest.php

```diff
@@ -76,8 +76,8 @@ public function testSetCustomValue()
 
     public function testConvertToJSon()
     {
+        $custom_values = [1, 2, 3];
```

Comment:
I think we don't need `$custom_values` variable because it used just one. Just put `[1,2,3]` as parameter.